apps/app/src/views/routes/Settings/components/profile/Profile.tsx

```diff
@@ -233,6 +235,12 @@ export const Profile = () => {
                   })
             }
           });
+          toast({
+            title: 'Role updated',
```

Comment:
We'll need this to be `Profile updated` instead of `Role updated`

Reply:
done